_layouts/contact.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
this is totally wrong.

Reply:
Please use jekyll format.

Reply:
I did this as materialize was conflicting with bootstrap.
I had to add another layout like default

Reply:
please refer to default layout. Everything is already there. 
Your file should start with
---
layout: default
---

Reply:
> I had to add another layout like default

Please don't do that.

> I did this as materialize was conflicting with bootstrap.

There is some reason for that.

Reply:
Ya i know and also triedthat.
But default layout mein tumne materiaize include Kiya h.. wo problem de rha h with bootstrap.
So i removed materiaize and it was working as expected.

Reply:
I searched for solutions on the internet but in vain. Everyone told not to use two frameworks together.

Reply:
> So i removed materiaize and it was working as expected.

Removing materialize isn't in the scope of this issue. The issue was only to add a contact page without changing the framework being used. 

Also I'm not sure but wouldn't using bootstrap instead of materialize cause problems with other pages that rely on that?

Whenever sending a PR, the changes should be made according to the frameworks and tech stack currently being used by that project 

If you want to change frameworks anyway a separate issue should be made for it and discussed by the members of the community since it is a big change.

I'm sure you can make the contact page work with materialize too :wink: 

Also lets stick to English as our medium of communication.